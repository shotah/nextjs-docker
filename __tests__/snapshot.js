/**
 * @jest-environment jsdom
 */

import React, { Suspense } from 'react';
import { render } from '@testing-library/react'
import { RecoilRoot } from 'recoil';
import Home from '../pages/index';

it('renders homepage unchanged', async() => { 
  const { asFragment } = render(
    <RecoilRoot>
      <Suspense fallback={<div>loading...</div>}>
        <Home />
      </Suspense>
    </RecoilRoot>
  );
  expect(asFragment()).toMatchSnapshot();
})

