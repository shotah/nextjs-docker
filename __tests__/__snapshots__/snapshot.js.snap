// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<DocumentFragment>
  <div
    class="md-container container"
  >
    <div
      class="container"
    >
      <h1>
         Welcome to Astro! 
      </h1>
      <p>
         Get started! 
      </p>
      <div>
        <h1>
          Count: 
          <span>
            0
          </span>
        </h1>
        <button
          class="btn btn-primary"
          type="button"
        >
          +1
        </button>
        <button
          class="btn btn-primary"
          type="button"
        >
          -1
        </button>
        <button
          class="btn btn-primary"
          type="button"
        >
          Reset
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
